source/get-started/bootable-usb.rst

```diff
@@ -88,11 +92,11 @@ Burn the |CL| image onto a USB drive
 
       sudo umount /dev/sdd*
 
-#. Burn the image onto the USB drive. The example below burns an image onto `<your USB device>`:
+#. Burn the image onto the USB drive. This example burns an image onto `/dev/sdd`.  The device name of the USB may vary.
```

Comment:
@mvincerx - would be better to say "Your USB device name may vary."?  Simply saying "The device name..." is too vague IMO.

Reply:
Sure. NP.

Reply:
@mvincerx - keep as is and ship it.  :)